src/wmnds/components/disruption-indicator/large/_example.njk

```diff
@@ -0,0 +1,13 @@
+{% raw %}
+{% from "wmnds/components/disruption-indicator/large/large.njk" import wmndsDisruptionIndicatorLarge %}
+
+{{
+  wmndsDisruptionIndicatorLarge({
+    primaryText: "Good service",
+    contentText: "A38",
+    classes: "wmnds-disruption-indicator-large--error",
+    iconLeft: "modes-isolated-bus",
+    iconRight: "general-warning-triangle"
```

Comment:
Ensure the ordering of the params is the same everywhere including the nunjucks properties.